src/webview/js/rewriteHtml.js

```diff
@@ -56,4 +55,46 @@ const rewriteImageUrls = (auth: Auth, element: Element | Document = document) =>
   });
 };
 
-export default rewriteImageUrls;
+/* Rewrite the 'time' elements under the given root, inclusive, to make them more readable.
+ *
+ * 1. Changes text to localized human readable date-time.
+ * 2. Adds 'original-text' attribrute, to show original text alert dialog on press.
+ */
+const rewriteTime = (element: Element | Document) => {
+  // Find the image elements to act on.
+  const timeElements = [].concat(
```

Comment:
When I hover over `elem` below, Flow reports the type:
> HTMLElement | HTMLTimeElement | any

I think that's the root of at least some of the need you're finding to add a conditional that should be impossible. In reality these elements should all be of type `HTMLTimeElement`.

Note how that's arranged in the other function:
```js
  const imageTags: $ReadOnlyArray<HTMLImageElement> = [].concat(
    element instanceof HTMLImageElement ? [element] : [],
    Array.from(element.getElementsByTagName('img')),
  );
```
with the type annotation. Try that here?